the-events-calendar.php

```diff
@@ -58,7 +58,7 @@ function tribe_events_not_php_version_plugin_name( $names ) {
 	}
 
 	add_filter( 'tribe_not_php_version_names', 'tribe_events_not_php_version_plugin_name' );
-	if ( ! has_filter( 'admin_notices', 'tribe_not_php_version_notice' ) ) {
+	if ( ! has_filter('admin_notices', 'tribe_not_php_version_notice' ) ) {
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening bracket; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis

<sub>hash: 7a09ef1a68c86937e148d5ff283a0cc8</sub>